genie-agent/src/main/java/com/netflix/genie/agent/cli/GlobalAgentArguments.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+import com.beust.jcommander.Parameter;
+import lombok.Getter;
+
+@Getter
+class GlobalAgentArguments {
+
+    @Parameter(names = {"-h", "--help"}, description = "Print help message", help = true)
```

Comment:
Not saying we shouldn't do this but this kind of tight coupling will break the abstraction provided by the `ArgParser` you made and almost makes it seem like there's no benefit to abstracting there at all.

Reply:
I'm not clear what coupling and which broken abstraction you're referring to.

Reply:
In the arg parser you mask that you're using this specific library behind APIs so we can swap out an implementation if need be. But here due to the presence of this specific annotation from that library we'd have to go through the code base and find equivalent functionality if we were to switch to another lib. It's not the biggest deal and it's more of a purity type thing but thought I'd point it out.